mypy_primer failure with mypy

```
UNEXPECTED FAILURE
        scripts/generate_matrix.py:1: error: disable_error_code: Invalid error code(s): any, decorated-any  [misc]
        Found 1 error in 1 file (checked 516 source files)
```

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -120,10 +120,12 @@ enable_error_code = [
     "redundant-expr",
     "truthy-bool",
 ]
-disallow_any_explicit = false # no other way to type e.g. `float64 <: number[Any]`
 warn_unreachable = false
 warn_unused_ignores = true
 # basedmypy/mypy compat
+disallow_any_decorated = false
+disallow_any_explicit = false
+disallow_any_expr = false
 disallow_subclassing_any = false
 # https://github.com/python/mypy/issues/18744
 disable_bytearray_promotion = true

--- a/scripts/generate_matrix.py
+++ b/scripts/generate_matrix.py
@@ -1,4 +1,3 @@
-# mypy: disable-error-code="any, decorated-any"
 # ruff: noqa: TRY003
 
 import importlib.metadata

